MODULE.bazel

```diff
@@ -1,22 +1,22 @@
-module(name = "com_github_google_benchmark", version="1.8.2")
+module(name = "google_benchmark", version="1.8.2")
```

Comment:
will this break anything anywhere?

Reply:
No, actually it is the other way round - in the Bazel Central Registry, the module name must match the directory, and the convention there is not to use 'com_github' prefixes. So the unified name now is 'google_benchmark'.